x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/FillMaskConfig.java

```diff
@@ -92,6 +107,8 @@ public String getWriteableName() {
     public void writeTo(StreamOutput out) throws IOException {
         vocabularyConfig.writeTo(out);
         out.writeNamedWriteable(tokenization);
+        out.writeVInt(numTopClasses);
```

Comment:
```suggestion
        numTopClasses = in.readInt();
```
We allow `-1` else where to indicate "Every top class", we should do the same here.